src/Chapter04/Listing04.44.DoWhileLoop.cs

```diff
@@ -16,7 +16,8 @@ public static void Main()
                 // Request a move from the current player
                 System.Console.Write(
                     $"\nPlayer {currentPlayer}: Enter move:");
-                string input = System.Console.ReadLine();
+                // TODO: Update listing in Manuscript
+                string input = System.Console.ReadLine() ?? string.Empty;
```

Comment:
For this one I think it might be cleaner to simply declare `input` and nullable
```suggestion
                string? input = System.Console.ReadLine();
```